frontend/src/components/Skill/Skill.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC, Suspense } from 'react';
+import { Box } from '@mui/material';
+
+import { LENGTH } from 'constants/sizes';
+import Loader from 'components/Loader';
+
+import SkillItem from './SkillItem/SkillItem';
+import { ISkillProps } from './types';
+import { SkillsTitle, SkillsBox, SkillsDivider } from './styled';
+
+const Skill: FC<ISkillProps> = ({ skillFounded, skills }) => {
+  const skillsData = skillFounded.length ? skillFounded : skills;
+  return (
+    <Box>
```

Comment:
Do you need this box just as a wrapper? maybe it's better to use an empty tag to avoid creating a new DOM node?